tasks/utils.py

```diff
@@ -64,7 +64,8 @@ def get_build_flags(ctx, static=False, use_embedded_libs=False):
         env["CGO_LDFLAGS_ALLOW"] = "-Wl,--allow-multiple-definition"
 
     if static:
-        ldflags += "-s -w -linkmode external -extldflags '-static' "
+        ldflags += "-s -w -linkmode=external '-extldflags=-static' "
+        env["CGO_ENABLED"] = "0"
```

Comment:
interested in why we need this too :)

Reply:
I believe this is due to the fact that `cgo` may implicitly turn on dynamic linking in some cases. I found this article interesting: https://www.osso.nl/blog/golang-statically-linked/